aten/src/ATen/core/jit_type.h

```diff
@@ -1566,9 +1566,16 @@ using ::torch::jit::Function;
 
 // Interfaces are a list of abstract methods that a class might meet.
 // If a class provides those methods, it implicitly meets the interface.
+// Subtype relations for Interface/ClassType
```

Comment:
I find this comment more confusing to understand than the actual code implementation. It would be better to explain that a class or interface `lhs` is a subtype of `rhs` if:

1. lhs methods are a superset of the methods of rhs
2. if rhs is a module interface, then lhs must be a module interface or module itself.